### DISCUSSION TITLE
Can't access logs in local

### DISCUSSION THREAD

**Discussion by ebenjs** [errsole/errsole.js#92] [Q&A]
Can anybody tell me if there is a way to access the logs in the local dashboard?

**Comment by mrrishimeena** [comment]
Did you visit http://localhost:8001/?

By default, the Errsole web dashboard can be accessed using this link.

Here is the documentation for accessing the web dashboard: https://github.com/errsole/errsole.js?tab=readme-ov-file#web-dashboard-access

**Reply by ebenjs** [reply]
@mrrishimeena I can access the Errsole web dashboard, but the logs are not displayed.

**Reply by mrrishimeena** [reply]
Can you tell me which versions of errsole.js and the storage module you are using?

**Reply by mrrishimeena** [reply]
Where are you running the app which is collecting the logs in the database? local or remote server?

**Reply by ebenjs** [reply]
I'm using `errsole-mongodb` version `errsole": "^2.9.1`. Here is a sample code of how i'm using it:

```typescript
import errsole from "errsole";
import ErrsoleMongoDB from "errsole-mongodb";

errsole.initialize({
  storage: new ErrsoleMongoDB(
    process.env.MONGODB_URI,
    'server_logs',
  ),
});
```

**Reply by ebenjs** [reply]
> Where are you running the app which is collecting the logs in the database? local or remote server?

I'm running the app in local but using a remote database for the logs. A MongoDB Atlas cluster database.

**Reply by mrrishimeena** [reply]
> > Where are you running the app which is collecting the logs in the database? local or remote server?
> 
> I'm running the app in local but using a remote database for the logs. A MongoDB Atlas cluster database.

Can we have a zoom call or google meet, where you can share the screen so we can understand this issue in detail?

**Reply by ebenjs** [reply]
Sure. Will you be available at 17:00 UTC ?

**Reply by mrrishimeena** [reply]
Sure, since we are already connected on LinkedIn, I will share the meeting link there 30 minutes before the meeting.

**Reply by ebenjs** [reply]
It is working fine now without any modification.
